pkg/apiserver/apiserver_test.go

```diff
@@ -931,7 +904,7 @@ func TestList(t *testing.T) {
 			legacy:    true,
 		},
 		{
-			url:       "/" + prefix + "/" + testGroupVersion.Group + "/" + testGroupVersion.Version + "/simple?namespace=other&labels=a%3Db&fields=c%3Dd",
+			url:       "/" + prefix + "/" + testGroupVersion.Group + "/" + testGroupVersion.Version + "/simple?namespace=other&labelSelector=a%3Db&fieldSelector=c%3Dd",
```

Comment:
No - it's not previous format - it's only the format that is introduced in the test - see ListOptions struct that I'm removing from addTestTypes(). In all production code we have labelSelector & fieldSelector.